src/Microsoft.DocAsCode.HtmlToPdf/PdfOptions.cs

```diff
@@ -41,13 +41,18 @@ public class PdfOptions
         /// <summary>
         /// Gets or sets the "Table of Contents" bookmark title.
         /// </summary>
-        public string TocTitle { get; set; } = "Cover Page";
+        public string TocTitle { get; set; } = "Table of Contents";
```

Comment:
I'd expect to keep the default value as "Cover Page", if user does not explicit provide a cover page. We can have a logic to change default value to "Table of Contents" if user provided cover page is found.